# Reading the data from JSON

In [17]:
import json

In [18]:
fd= open('record.json','r')
r=fd.read()
fd.close()

record=json.loads(r)



In [19]:
record

{'1000': {'Name': 'Chococake',
  'price': 20,
  'Quantity': 40,
  'Company': 'zzz',
  'Best_before': '3 months',
  'Category': 'pastry'},
 '1001': {'Name': 'Vanillacake',
  'price': 20,
  'Quantity': 40,
  'Company': 'zzz',
  'Best_before': '3 months',
  'Category': 'pastry'},
 '1002': {'Name': 'Pistacake',
  'price': 20,
  'Quantity': 40,
  'Company': 'zzz',
  'Best_before': '3 months',
  'Category': 'pastry'},
 '1003': {'Name': 'plumcake',
  'price': 20,
  'Quantity': 40,
  'Company': 'zzz',
  'Best_before': '3 months',
  'Category': 'pastry'},
 '1004': {'Name': 'Bananacake',
  'price': 20,
  'Quantity': 40,
  'Company': 'zzz',
  'Best_before': '3 months',
  'Category': 'pastry'},
 '1005': {'Name': 'Coke',
  'price': 30,
  'Quantity': 50,
  'Company': 'zzz',
  'Best_before': '5 months',
  'Category': 'drinks'},
 '1006': {'Name': 'Fanta',
  'price': 30,
  'Quantity': 50,
  'Company': 'zzz',
  'Best_before': '5 months',
  'Category': 'drinks'},
 '1007': {'Name': 'Limca',
  'price': 30,

# Adding new products into Inventory

In [21]:
prod_id=str(input('Enter the product Id: '))
prod_name= str(input("Enter the product's name: "))
prod_price=int(input('Enter the price: '))
prod_quan=int(input('Enter the quantity: '))
prod_comp=str(input("Enter the product's company: "))
prod_bb=str(input('Enter the product best before in months: '))
prod_cat=str(input("Enter the product's category: "))

record[prod_id]={'Name':prod_name, 'price':prod_price,'Quantity':prod_quan,'Company':prod_comp,'Best_before':prod_bb,'Category':prod_cat}

js= json.dumps(record)

fd= open('record.json','w')
fd.write(js)
fd.close()

Enter the product Id: 1011
Enter the product's name: thumbsup
Enter the price: 40
Enter the quantity: 100
Enter the product's company: zzz
Enter the product best before in months:  3 months
Enter the product's category: drinks


# Updated record with the new item

In [24]:
record

{'1000': {'Name': 'Chococake',
  'price': 20,
  'Quantity': 40,
  'Company': 'zzz',
  'Best_before': '3 months',
  'Category': 'pastry'},
 '1001': {'Name': 'Vanillacake',
  'price': 20,
  'Quantity': 40,
  'Company': 'zzz',
  'Best_before': '3 months',
  'Category': 'pastry'},
 '1002': {'Name': 'Pistacake',
  'price': 20,
  'Quantity': 40,
  'Company': 'zzz',
  'Best_before': '3 months',
  'Category': 'pastry'},
 '1003': {'Name': 'plumcake',
  'price': 20,
  'Quantity': 40,
  'Company': 'zzz',
  'Best_before': '3 months',
  'Category': 'pastry'},
 '1004': {'Name': 'Bananacake',
  'price': 20,
  'Quantity': 40,
  'Company': 'zzz',
  'Best_before': '3 months',
  'Category': 'pastry'},
 '1005': {'Name': 'Coke',
  'price': 30,
  'Quantity': 50,
  'Company': 'zzz',
  'Best_before': '5 months',
  'Category': 'drinks'},
 '1006': {'Name': 'Fanta',
  'price': 30,
  'Quantity': 50,
  'Company': 'zzz',
  'Best_before': '5 months',
  'Category': 'drinks'},
 '1007': {'Name': 'Limca',
  'price': 30,

#### here above we can see our new item thumbsup added as 1011